tests/rest/admin/test_user.py

```diff
@@ -2049,11 +2063,113 @@ def test_get_media(self):
+        media1 = self._create_media_and_access(other_user_tok, image_data1, "image.png")
+        self.pump(1.0)
+        media2 = self._create_media_and_access(other_user_tok, image_data2, "image.gif")
+        self.pump(1.0)
+        media3 = self._create_media_and_access(other_user_tok, image_data3, "image.bmp")
+        self.pump(1.0)
+
+        # Mark one media as safe from quarantine.
+        self.get_success(self.store.mark_local_media_as_safe(media2))
+        # Quarantine one media
+        self.get_success(
+            self.store.quarantine_media_by_id("test", media3, self.admin_user)
+        )
+
+        # order by default ("created_ts")
+        # default is backwards
+        self._order_test(None, [media3, media2, media1])
+        self._order_test(None, "f", [media1, media2, media3])
+        self._order_test(None, "b", [media3, media2, media1])
+
+        # sort by media_id
+        sorted_media = sorted([media1, media2, media3], reverse=False)
+        sorted_media_reverse = sorted(sorted_media, reverse=True)
+
+        # order by media_id
+        self._order_test("media_id", sorted_media)
+        self._order_test("media_id", "f", sorted_media)
+        self._order_test("media_id", "b", sorted_media_reverse)
+
+        # order by upload_name
+        self._order_test("upload_name", [media3, media2, media1])
+        self._order_test("upload_name", "f", [media3, media2, media1])
+        self._order_test("upload_name", "b", [media1, media2, media3])
+
+        # order by media_type
+        # result is ordered by media_id
+        # because of uploaded media_type is always 'application/json'
+        self._order_test("media_type", sorted_media)
+        self._order_test("media_type", "f", sorted_media)
+        self._order_test("media_type", "b", sorted_media)
+
+        # order by media_length
+        self._order_test("media_length", [media2, media3, media1])
+        self._order_test("media_length", "f", [media2, media3, media1])
+        self._order_test("media_length", "b", [media1, media3, media2])
+
+        # order by created_ts
+        self._order_test("created_ts", [media1, media2, media3])
+        self._order_test("created_ts", "f", [media1, media2, media3])
+        self._order_test("created_ts", "b", [media3, media2, media1])
+
+        # order by last_access_ts
+        self._order_test("last_access_ts", [media1, media2, media3])
+        self._order_test("last_access_ts", "f", [media1, media2, media3])
+        self._order_test("last_access_ts", "b", [media3, media2, media1])
+
+        # order by quarantined_by
+        # one media is in quarantine, others are ordered by media_ids
+        # Different sort order of SQlite and PostreSQL foor bool
+        # self._order_test("quarantined_by", sorted([media1, media2]) + [media3])
+        # self._order_test("quarantined_by", "f", sorted([media1, media2]) + [media3])
+        # self._order_test("quarantined_by", "b", [media3] + sorted([media1, media2]))
```

Comment:
This seems unfortunate. Why is this not the case for `safe_from_quarantine`?

Reply:
I will do a new test.

Reply:
Different sort order of SQlite and PostreSQL:
If a media is not in quarantine `quarantined_by` is NULL
SQLite considers NULL to be smaller than any other value.
PostreSQL considers NULL to be larger than any other value.